build.gradle

```diff
@@ -250,6 +250,20 @@ void fail(String msg) {
     throw new GradleException("FAIL: " + msg);
 }
 
+/**
+ * Converts the given OS architecture to unified form
+ *
+ * @param arch OS architecture
+ * @return unified OS architecture which is used for JavaFX building
+ */
+String getConvertedOsArch(String arch) {
+    switch (arch) {
+        case "aarch64" : return "arm64"
+        case "amd64" : return "x64"
+        default: return arch
+    }
```

Comment:
I think a function that gets the "converted" arch is very useful, but I'm not sure what the resulting value should be. In case of arm64/aarch64, I have s slight preference for aarch64, but I'm not against arm64.

Reply:
This does seem useful, although in this case, wasn't `arm64` chosen for compatibility with the name in the Microsoft VisualStudio distro? If so, then this part of the most recent change might be Windows-specific. It might be better to revert this (going back to your previous fix in `win.gradle`) until we can take a more holistic look at this.

Reply:
Yes, the arm64 is used as an OS arch for Visual Studio vcvarsall.bat and x64 as a part of a path to arch specific Visual Studio tools on Windows.

I reverted the fix with converted os arch variables back.